src/main/groovy/org/pih/warehouse/core/DocumentCode.groovy

```diff
@@ -25,7 +25,6 @@ enum DocumentCode {
 
     static list() {
         [
-                SHIPPING_TEMPLATE,
```

Comment:
why has it been removed from there? isn't it used anywhere else?

Reply:
there were two `SHIPPING_TEMPLATE` in the `list()` figured that was a mistake and removed one of them

Reply:
normally I'd say we shouldn't do this change since there's no direct bug associated with it and we're already mostly through testing but I don't actually see this `list()` call being used anywhere so likely it's fine.